### PR TITLE
p0-card-images-and-input compo

### PR DESCRIPTION
**Pull request by sumitk-singh**
waiting for the code review.

### CODE REVIEW

File: components/Card.css [hunk 1]
```diff
@@ -0,0 +1,261 @@
+/* 1. Cards with badges */
+
+.card-with-badge {
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    height: 18.9rem;
+    width: 12.6rem;
+    background-color: #ffd9c9;
```
Comment (Omkar-Ghate) [inline components/Card.css]: maybe you can use var for colors

File: components/Card.css [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+    text-align: center;
+    background: white;
+}
+
+.card-with-badge>p,
+.card-with-dismiss>p {
+    border-radius: 15px;
+    transform: rotate(-45deg);
+    text-align: center;
+    padding: 0.315rem;
```
Comment (Omkar-Ghate) [inline components/Card.css]: use px for values below 10px

File: components/Card.css [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+/* 5. vertical card w/ and w/o shadow */
+
+.vertical-card-example {
+    display: flex;
+    flex-direction: row;
+    gap: 1rem;
+}
+
+.vertical-card-example>h3 {
+    line-height: 0.05rem;
```
Comment (Omkar-Ghate) [inline components/Card.css]: can use px

File: components/Card.css [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+.horizontal-card-with-shadow,
+.horizontal-card-without-shadow,
+.horizontal-card-with-text-overlay {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    position: relative;
+    width: 37.8rem;
+    height: auto;
+    background: #ffd9c9;
```
Comment (Omkar-Ghate) [inline components/Card.css]: can use var for colors

**Review by Omkar-Ghate — CHANGES_REQUESTED** [review]
Use px for smaller values. Rest everything looks good for me.

File: components/Card.css [hunk 5]
```diff
@@ -0,0 +1,261 @@
...
+
+.coupon-off {
+    font-size: 3rem;
+    color: #ff4800;
+}
+
+.card-close-btn {
+    color: gray;
+    padding: 0.315rem;
+    margin-top: 0.315rem;
```
Comment (sudipkundu999) [inline components/Card.css]: Try to use px below 1rem

File: components/Card.css [hunk 6]
```diff
@@ -0,0 +1,261 @@
...
+    width: 37.8rem;
+    height: auto;
+    background: #ffd9c9;
+}
+
+.horizontal-card-with-shadow>img,
+.horizontal-card-without-shadow>img,
+.horizontal-card-with-text-overlay>img {
+    height: auto;
+}
```
Comment (sudipkundu999) [inline components/Card.css:232]: "height" has a default value of auto

**Review by sudipkundu999 — APPROVED** [review]
Try to use variables for colors.
Good to merge with changes

File: components/Card.css [hunk 7]
```diff
@@ -0,0 +1,261 @@
...
+    border: dashed #ff4800;
+    margin-top: 1.575rem;
+    text-align: center;
+    background: white;
+}
+
+.card-with-badge>p,
+.card-with-dismiss>p {
+    border-radius: 15px;
+    transform: rotate(-45deg);
```
Comment (srishtigautam21) [inline components/Card.css:58]: use rem for values above 10px

**Review by srishtigautam21 — COMMENTED** [review]
do the changes in px, rem and colors as suggested

File: components/Card.css [hunk 8]
```diff
@@ -0,0 +1,261 @@
...
+}
+
+.card-with-dismiss>.coupon-heading {
+    padding-top: auto;
+    text-align: center;
+}
+
+.coupon-code {
+    position: relative;
+    border: dashed #ff4800;
```
Comment (deekshithmd) [inline components/Card.css]: try to use color variables

File: components/Card.css (hunk 2 above)
Comment (sumitk-singh) [reply components/Card.css]: OK

**Review by Omkar-Ghate — APPROVED** [review]
Good to merge.

**Review by srishtigautam21 — APPROVED** [review]

**Pull request merged**